src/sidebar/store/create-store.js

```diff
@@ -187,6 +187,22 @@ export function createStore(modules, initArgs = [], middleware = []) {
   return store;
 }
 
+/**
+ * Helper for creating an action which checks that the type of the action's
+ * payload is compatible with what the reducer expects.
```

Comment:
I suggest looking at changes in the store module first to understand how this is used.